File: samples/android/clients/VirtualAssistantClient/app/src/main/java/com/microsoft/bot/builder/solutions/virtualassistant/models/OpenDefaultApp.java
package com.microsoft.bot.builder.solutions.virtualassistant.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class OpenDefaultApp {
    @Expose
    @SerializedName("meetingUri")
    public String meetingUri;

    @Expose
    @SerializedName("telephoneUri")
    public String telephoneUri;

    @Expose
    @SerializedName("mapsUri")
    public String mapsUri;

    @Expose
    @SerializedName("musicUri")
    public String musicUri;
}

File: sdk/csharp/libraries/microsoft.bot.builder.solutions/Models/OpenDefaultApp.cs
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

using System;

namespace Microsoft.Bot.Builder.Solutions.Models
{
    [Obsolete("This type is being deprecated. It's moved to the assembly Microsoft.Bot.Solutions. Please refer to https://aka.ms/botframework-solutions/releases/0_8", false)]
    public class OpenDefaultApp
    {
        public string MeetingUri { get; set; }

        public string TelephoneUri { get; set; }

        public string MapsUri { get; set; }

        public string MusicUri { get; set; }
    }
}
